### ISSUE TITLE
incoming/tags.txt missing

### ISSUE THREAD

**Issue by TinoDidriksen** [giellalt/lang-nno#1]
`No rule to make target 'incoming/tags.txt', needed by 'root.lexc'.`

**Comment by Trondtr** [comment]
Here for lang-nno, just as for Tino's similar report on lang-esu, there was (despite the error message, which I was able to repeat) such a rule in src/fst/Makefile.am.

What I then did was to write
touch src/fst/incoming/tags.txt
touch src/fst/incoming/entries.lexc

(the second line as a response to the error message coming after the first command). Thereafter, nno compiled. 

Now, this is not how we would like it to be. Whether the makefile should include a touch command or whether we should create and check in (empty?) files in incoming I do not know, but as it is now, it stops at the error message reported by Tino.
